docs/hazmat/primitives/symmetric-encryption.rst

```diff
@@ -35,9 +29,12 @@ in an "encrypt-then-MAC" formulation as `described by Colin Percival`_.
 
     .. doctest::
 
+        >>> import os
         >>> from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
         >>> from cryptography.hazmat.backends import default_backend
         >>> backend = default_backend()
+        >>> key = os.urandom(32)
+        >>> iv = os.urandom(32)
```

Comment:
IV should actually be 16 (the block size of AES is 16 bytes). This should fail in the `validate_for_algorithm` check, but we won't know until Travis finishes up :)